compliance_client/__init__.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+
+from sys import exit,stderr
+from insights.client.connection import InsightsConnection
+from insights.client.config import InsightsConfig
+from insights.util.canonical_facts import get_canonical_facts
+from subprocess import Popen, PIPE, STDOUT
+from re import findall
+from os import environ
+from platform import linux_distribution
+from glob import glob
+
+OSCAP_RESULTS_OUTPUT = '/tmp/oscap_results.xml'
+UPLOAD_ARGS = filter(None, [environ.get('UPLOAD_ARGS', None)])
+NONCOMPLIANT_STATUS = 2
```

Comment:
oscap returns 1 if the scan fails. It returns 2 if the scan succeeds but the host is non-compliant.